src/CMakeLists.txt

```diff
@@ -48,12 +48,13 @@ ENDIF()
 # Main library.
 ADD_LIBRARY(roboptim-core SHARED ${HEADERS} ${ROBOPTIM_CORE_SRC})
 TARGET_COMPILE_OPTIONS(roboptim-core PUBLIC "-Droboptim_core_EXPORTS")
-PKG_CONFIG_USE_DEPENDENCY(roboptim-core eigen3)
 PKG_CONFIG_USE_DEPENDENCY(roboptim-core liblog4cxx)
 
 # Add required libs to pkg-config file.
 SET(ROBOPTIM_API_BOOST_LIBRARIES date_time system filesystem)
-PKG_CONFIG_APPEND_BOOST_LIBS(${ROBOPTIM_API_BOOST_LIBRARIES})
+IF(NOT WIN32)
+  PKG_CONFIG_APPEND_BOOST_LIBS(${ROBOPTIM_API_BOOST_LIBRARIES})
+ENDIF(NOT WIN32)
```

Comment:
I understand, but on windows, having the include and the link folder is sufficient for the automatically linking.
Besides, the name of the libs is not simply boostxxx.lib, but boostxxx-vcCompilerVersion-multithread-BoostVersion.lib, so doing it manually would be bothersome